src/utils/presence.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+
+const OFFLINE_THRESHOLD_SECS = 140;
+
+export const getAggregatedPresence = (presence: Object) => {
```

Comment:
our `statusFromPresence` is in `userHelper`, so should I move this too in that?
I thinks its better to have it separately. Thoughts?